content/en/dashboards/sharing.md

```diff
@@ -71,7 +71,7 @@ To revoke a shared dashboard:
 
 ### Applying restrictions
 
-You can restrict access on a per IP address basis to your dashboard. Email [Datadog support][6] to enable the IP address white listing feature that allows administrators to provide a list of IP addresses that have access to shared dashboards. Once enabled, manage your restrictions on your organization's [security page][7].
+You can restrict access on a per IP address basis to your dashboard. Email [Datadog support][6] to enable the IP address pass listing feature that allows administrators to provide a list of IP addresses that have access to shared dashboards. Once enabled, manage your restrictions on your organization's [security page][7].
```

Comment:
```suggestion
You can restrict access on a per IP address basis to your dashboard. Email [Datadog support][6] to enable the IP address include listing feature that allows administrators to provide a list of IP addresses that have access to shared dashboards. Once enabled, manage your restrictions on your organization's [security page][7].
```

Reply:
Thanks for the update! We're currently part of an internal guild working toward these changes, but this is the language we've landed on so far for updates.